release/src/main/scripts/verify_release_build.sh

```diff
@@ -123,12 +123,15 @@ if [[ ! -z `which hub` ]]; then
   # The version change is needed for Dataflow python batch tests.
   # Without changing to dev version, the dataflow pipeline will fail because of non-existed worker containers.
   # Note that dataflow worker containers should be built after RC has been built.
-  sed -i -e "s/${RELEASE_VER}/${RELEASE_VER}.dev/g" sdks/python/apache_beam/version.py
-  sed -i -e "s/sdk_version=${RELEASE_VER}/sdk_version=${RELEASE_VER}.dev/g" gradle.properties
+  sed -i -e "s/${RELEASE_VER}\(.dev\)\?/${RELEASE_VER}.dev/g" sdks/python/apache_beam/version.py
```

Comment:
It would be even better to replace this with a call to [`set_version.sh`](https://github.com/apache/beam/blob/master/release/src/main/scripts/set_version.sh)